Jenkinsfile

```diff
@@ -15,14 +15,27 @@ def mavenEnv(Map params = [:], Closure body) {
   retry(count: attempts, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
     echo 'Attempt ' + ++attempt + ' of ' + attempts
     // no Dockerized tests; https://github.com/jenkins-infra/documentation/blob/master/ci.adoc#container-agents
-    node(params['nodePool'] ? 'maven-bom': 'maven-' + params['jdk']) {
+    node('maven-bom') {
```

Comment:
- From infra: it is easier for us to observe bom builds if all agents are in the same label system
- Also required as only BOM agents has have access to the cache volume

Reply:
OK since https://github.com/jenkinsci/bom/blob/ffe8b6281366ec63337495b14beb31a550f9e13e/Jenkinsfile#L23 should still apply.

Reply:
absolutely, thanks for making it explicitly (I forgot with the head in the changes) as it is important!